G/GDAL/build_tarballs.jl

```diff
@@ -132,20 +135,20 @@ hdf5_platforms = expand_cxxstring_abis(hdf5_platforms)
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("GEOS_jll"; compat="~3.11"),
-    Dependency("PROJ_jll"; compat="~900.100"),
+    Dependency("GEOS_jll"; compat="3.11.2"),
```

Comment:
This looks fine to me. GEOS 3.12 is available now, which this new compat will allow, because GEOS 3.12 is not breaking.

Right now GDAL is holding back GEOS to ~3.11.